include/s3select_oper.h

```diff
@@ -1434,27 +1434,39 @@ class arithmetic_operand : public base_statement
     switch (_cmp)
     {
     case cmp_t::EQ:
-      return var_value =  bool( (l->eval() == r->eval()) ^ negation_result );
```

Comment:
maybe, its better to change value::bool(), instead.
it will be transparent.

Reply:
Thanks. That looks good.